### ISSUE TITLE
Is there a way to have <strong>, <b>, <em>, <i> and <u> tags?

### ISSUE THREAD

**Issue by alexmsantos** [Quartz/aml-gdoc-server#6]
It would be great to have these tags parsed as well...

**Comment by yanofsky** [comment]
Agreed! But this tool doesn't do the Archie parsing on its own. It uses `doc-to-archieml`

See the discussion of this issue over at that repo https://github.com/rdmurphy/doc-to-archieml/issues/4
